sim/battle.ts

```diff
@@ -286,6 +286,12 @@ export class Battle {
 		this.prng = new PRNG(this.prng.startingSeed);
 	}
 
+	reseed(newSeed?: [number, number, number, number]) {
```

Comment:
We already have a `resetRNG` function which could have been used for this. We also have a `PRNGSeed` type so you don't need to write out the entire tuple.